setup.py

```diff
@@ -30,5 +36,20 @@
           'externals': InstallExternals,
           'help': Help,
       },
-      package_data={'fitbenchmarking': ['resproc/color_definitions.txt']}
+      package_data={'fitbenchmarking': ['utils/default_options.ini']}
```

Comment:
I'm not sure what the distribution plan is for fitbenchmarking, but if we are going to make both source and built distributions, it is worth noting that apparently `package_data` [only includes files for built distributions](https://stackoverflow.com/questions/7522250/how-to-include-package-data-with-setuptools-distribute).

Reply:
Neither am I... but thanks for the pointer! It looks like problems in that thread are only relevant in the case of having a "MANIFEST.in" file? So this won't be a problem for us (until we add one).

Do you have any recommendations for alternatives, I see that there are a few ways people are claiming this should be done...?

Reply:
I think actually it suggests that the MANIFEST.in file is the way to include files for source distributions, so if we are going to have both source and built distributions we might need to include this as well.  This probably isn't the best forum for this, just thought I'd mention it!

Reply:
After some more digging I found that if there's no MANIFEST.in file, setuptools will add the package_data to the MANIFEST file. (https://docs.python.org/2/distutils/setupscript.html#installing-package-data)

Reply:
Ah excellent, well discovered!